cves/CVE-2009-2901.yml

```diff
@@ -139,7 +152,11 @@ interesting_commits:
     emerging themes?
     If there are no interesting commits, demonstrate that you completed this
     section by explaining what happened between the VCCs and the fix.
-  answer:
+  answer: |
+    Not really. There were a handful of bugfixes, a lot of updates to STATUS.txt
+    (which seems to be just tracking progress of existing bug fixes), but no real
+    changes to anything that would touch deployment. This bug was likely in the
+    system for a good while.
```

Comment:
Perhaps rephrase last sentence to be less conversational

Reply:
I generally write in a fairly casual way, which may not necessarily reflect the nature of the assignment. I'll rewrite it to make it sound a little more professional.